### PR TITLE
single Product listing created

### PR DESCRIPTION
**Pull request by ivinayakg**
single page product listing.
Web capture_30-3-2022_145034_vinayakecom netlify app

### CODE REVIEW

File: src/components/css/SingleProductPage.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.productPage_discount {
+  font-size: 1.8rem;
+  text-decoration: line-through;
+  text-decoration-color: rgba(0, 0, 0, 0.3);
+  color: rgba(0, 0, 0, 0.35);
+}
+
+.product_Qauntity {
```
Comment (bhavyaj12) [inline src/components/css/SingleProductPage.css:42]: Quantity spelling can be corrected. I'm not sure about the naming convention of the variables it seems a little inconsistent?

**Review by bhavyaj12 — APPROVED** [review]
Code is easy to understand and works well.
UI is clean. Navbar cart wishlist badge seems out of place, that's it.

File: src/components/SingleProductPage.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <div className="productPage_detail">
+          <h2 className="head2">{data.name}</h2>
+          <h4 className="head4 text-fade">{data.brand}</h4>
+          <div className="productPage_price mb1">
+            <h2 className="head2">
+              <span>Rs.</span> {data.price - (data.discount / 100) * data.price}
+            </h2>
+            <h2 className="head2 productPage_discount">
+              <span>Rs.</span> {data.price}
+            </h2>
```
Comment (viru2001) [inline src/components/SingleProductPage.js:35]: you can destructure data object at start

File: src/components/SingleProductPage.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    dispatch({ for: "wishlist", type: "add", payload: data });
+  };
+
+  const addToCart = () => {
+    dispatch({ for: "cart", payload: data, type: "add" });
+    notificationHandler({
+      type: "success",
+      content: "Succesfully Added to cart",
+    });
+  };
```
Comment (viru2001) [inline src/components/SingleProductPage.js:18]: you can add these functions in utils

**Review by viru2001 — APPROVED** [review]
The code is good and easy to understand.

**Pull request merged**